sponge/src/main/java/com/griefdefender/listener/PlayerEventHandler.java

```diff
@@ -1231,7 +1252,7 @@ private void onPlayerHandleClaimCreateAction(InteractEvent event, BlockSnapshot
             Chunk chunk = player.getWorld().getChunk(location.getBlockX() >> 4, 0, location.getBlockZ() >> 4).get();
             int miny = location.getBlockY();
             World world = chunk.getWorld();
-            final Chunk newChunk = world.regenerateChunk(chunk.getPosition().getX(), 0, chunk.getPosition().getZ()).orElse(null);
+            //final Chunk newChunk = world.regenerateChunk(chunk.getPosition().getX(), 0, chunk.getPosition().getZ()).orElse(null);
```

Comment:
This should not be changed.

Reply:
Right, my IDE was messing up and couldn't resolve that exact method and I forgot to look into it